components/header/header.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import style from './header.module.scss';
+
+export default function Header() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <div className={style.container}>
+      <div className={style.header}>
+        <a href="#" className={style.logo}>
+          <Image
+            alt="KedroLogo"
+            src="/images/kedro-logo.svg"
+            layout="intrinsic"
+            width={30}
+            height={30}
+          />
+          <h4 className={style.logoText}>Kedro</h4>
+        </a>
+        {/*TO DO - Update All Links*/}
+        <nav
+          className={menuOpen ? `${style.menu} ${style.active}` : style.menu}
+          onClick={() => setMenuOpen(false)}
+        >
+          <a href="#why-kedro" className={style.link}>
+            Why Kedro?
+          </a>
+          <a href="#features" className={style.link}>
+            Features
+          </a>
+          <a href="#" className={style.link}>
+            Get Started
+          </a>
+          <a href="#" className={style.link}>
+            Documentation
+          </a>
```

Comment:
I think we have the link for this. You can use: https://kedro.readthedocs.io/en/stable/